src/codecs/avif/mod.rs

```diff
@@ -23,12 +24,33 @@ use rgb::AsPixels;
 pub struct AvifEncoder<W> {
     inner: W,
     fallback: Vec<u8>,
+    config: Config
 }
 
 impl<W: Write> AvifEncoder<W> {
     /// Create a new encoder that writes its output to `w`.
     pub fn new(w: W) -> Self {
-        AvifEncoder { inner: w, fallback: vec![] }
+        AvifEncoder::new_with_speed_quality(w, 1, 100)
+    }
+
+    /// Create a new encoder with specified speed and quality,
+    /// that writes its output to `w`.
+    pub fn new_with_speed_quality(w: W, speed: u8, quality: u8) -> Self {
```

Comment:
Could we document the allowed ranges for quality and speed?

Reply:
Absolutely. I've added that now.